spec.bs

```diff
@@ -136,3 +136,115 @@ dictionary URLPatternComponentResult {
   record<USVString, USVString> groups;
 };
 </xmp>
+
+Each {{URLPattern}} object has an associated <dfn for=URLPattern>protocol component</dfn>, a <a for=/>component</a>, initially null.
```

Comment:
If you want you can use `[=/component=]` here and below; `<a>` should never really be necessary.

Reply:
Rewrote to use `[=component=]`.  Note, I didn't seem to need that slash in there.